src/bundle/views/BundleOverview.tsx

```diff
@@ -0,0 +1,316 @@
+					return props;
+				});
+				return userProfiles && <AvatarList avatars={avatarProps} isOpen={false} />;
+
+			case 'actions':
+				return renderActions(id);
+
+			case 'created_at':
+			case 'updated_at':
+				const cellData = collection[colKey as 'created_at' | 'updated_at'];
+				return <span title={formatTimestamp(cellData)}>{fromNow(cellData)}</span>;
+
+			default:
+				return null;
+		}
+	};
+
+	const renderTable = (bundles: Avo.Collection.Collection[]) => (
+		<>
+			<Table
+				columns={[
+					{ id: 'thumbnail', label: '', col: '2' },
+					{
+						id: 'title',
+						label: t('collection/views/collection-overview___titel'),
+						col: '6',
+						sortable: true,
+					},
+					{
+						id: 'updated_at',
+						label: t('collection/views/collection-overview___laatst-bewerkt'),
+						col: '3',
+						sortable: true,
+					},
+					{ id: 'access', label: t('collection/views/collection-overview___toegang'), col: '2' },
+					{ id: 'actions', label: '', col: '1' },
+				]}
+				data={bundles}
+				emptyStateMessage={t('collection/views/collection-overview___geen-resultaten-gevonden')}
+				renderCell={renderCell}
+				rowKey="id"
+				variant="styled"
+				onColumnClick={onClickColumn as any}
+				sortColumn={sortColumn}
+				sortOrder={sortOrder}
+			/>
+			<Pagination
+				pageCount={Math.ceil(numberOfBundles / ITEMS_PER_PAGE)}
+				currentPage={page}
+				onPageChange={setPage}
+			/>
+		</>
+	);
+
+	const renderEmptyFallback = () => (
+		<ErrorView icon="folder" message={t('Je hebt nog geen bundels aangemaakt')}>
+			<p>
+				<Trans>
+					Een bundel is een verzameling van collecties rond een bepaald thema of voor een bepaalde
+					les. Nadat je een bundel hebt aangemaakt kan je deze delen met andere gebruikers om samen
+					aan te werken. Andere gebruikers kunnen ook bundels met jou delen die je dan hier
+					terugvindt.
+				</Trans>
+			</p>
+			<Spacer margin="top">
+				<Button
+					type="primary"
+					icon="search"
+					label={t('Zoek een collectie en maak je eerste bundel')}
+					onClick={onClickCreate}
+				/>
+			</Spacer>
+		</ErrorView>
+	);
+
+	const renderCollections = (
+		collections: Avo.Collection.Collection[],
+		refetchCollections: () => void
+	) => (
+		<>
+			{collections.length ? renderTable(collections) : renderEmptyFallback()}
+			<DeleteObjectModal
+				title={t('collection/views/collection-overview___verwijder-collectie')}
+				body={t(
+					'collection/views/collection-overview___bent-u-zeker-deze-actie-kan-niet-worden-ongedaan-gemaakt'
+				)}
+				isOpen={isDeleteModalOpen}
+				onClose={() => setIsDeleteModalOpen(false)}
+				deleteObjectCallback={() => onDeleteCollection(refetchCollections)}
+			/>
+		</>
+	);
+
+	return (
+		<DataQueryComponent
+			query={GET_COLLECTIONS_BY_OWNER}
+			variables={{
+				owner_profile_id: getProfileId(user),
+				offset: page * ITEMS_PER_PAGE,
+				limit: ITEMS_PER_PAGE,
+				order: { [sortColumn]: sortOrder },
+				type_id: 4,
```

Comment:
Eventueel een enum maken voor die `type_id`'s om duidelijk te maken wat je opvraagt?

Reply:
die bestond blijkbaar al: ContentTypeNumber.bundle
fixed in: https://github.com/viaacode/avo2-client/commit/430c85dc9b87545c9332184b0c22dd8cbef01dc9